src/components/qr-code/qr-code.component.ts

```diff
@@ -51,7 +51,8 @@ export default class SlQrCode extends ShoelaceElement {
       return;
     }
 
-    QrCreator.render(
+    // For some reason, when changing to "NodeNext", it has the wrong type for QrCreator.
+    ;(QrCreator as unknown as typeof QrCreator.default).render(
```

Comment:
This ugly hack exists because the only way to reliably check imports have extensions is to set `"moduleResolution": "NodeNext"`, which means needing to set `"module": "NodeNext"`, which then causes this to get the wrong file + type definition.

Issue here: https://github.com/microsoft/TypeScript/issues/42813